CMakeLists.txt

```diff
@@ -91,6 +91,14 @@ if(NOT K2_WITH_CUDA)
   set(K2_ENABLE_NVTX OFF CACHE BOOL "" FORCE)
 endif()
 
+if(K2_WITH_CUDA)
+  find_package(CUDA QUIET)
+  message(STATUS "CUDA_INCLUDE_DIRS=${CUDA_INCLUDE_DIRS}")
+  message(STATUS "CUDA_LIBRARIES=${CUDA_LIBRARIES}")
+  message(STATUS "CUDA_nvToolsExt_LIBRARY=${CUDA_nvToolsExt_LIBRARY}")
+  message(STATUS "CUDA_TOOLKIT_ROOT_DIR=${CUDA_TOOLKIT_ROOT_DIR}")
+endif()
+
```

Comment:
Is it necessary?

I think the following line
https://github.com/k2-fsa/k2/blob/1e22c77e9cc76648eb34a9f7a265fc8bc7c9c6d5/CMakeLists.txt#L48
introduces support for CUDA.

please have a look at
https://cliutils.gitlab.io/modern-cmake/chapters/packages/CUDA.html

Reply:
My local cmake version is  3.16.0，maybe the feature need version 3.17 or 3.18 or latest.
And CUDA_nvToolsExt_LIBRARY is needed by K2_WITH_NVTX.